### PR TITLE
fix(#2410): broken rounded corners for dropdown in input group

### PR DESCRIPTION
**Pull request by illiteratewriter**
- [x] Bug fix 
- [ ] New feature 
- [ ] Chore 
- [ ] Breaking change 
- [x] There is an open issue which this change addresses
- [x] I have read the **[CONTRIBUTING](./CONTRIBUTING.md)** document.
- [x] My commits follow the [Git Commit Guidelines](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#commits)
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly.
- [ ] My change requires a change to [Typescript typings](./types/lib).
  - [ ] I have updated the typings accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by illiteratewriter** [comment]
The issue was the selector for dropdown in the input group changed for the new version of Bootstrap. The new selector selected the immediate child of InputGroup, whereas the previous version had a div between InputGroup and the Button.

I removed the extra `div` but then all of the remaining code was dependent on selecting via that particular `div`. So have made changes to fix them.

**Comment by davidacevedo** [comment]
I like the format you were doing before for commits, `fix(#1410): your commit message here`

File: src/Dropdown.js [hunk 1]
```diff
@@ -101,14 +110,18 @@ class Dropdown extends React.Component {
     return this.menuRef.current;
   }
 
+  getToggle() {
+    return this.toggleRef.current;
+  }
```
Comment (davidacevedo) [inline src/Dropdown.js:115]: I don't think we need this method as on line `119` we are not using this.

Comment (illiteratewriter) [reply src/Dropdown.js:115]: We are using that on line `154`, so added to `119` as well.

**Comment by davidacevedo** [comment]
Did this PR get a bit broken? It seems to have a lot more changes now?

**Comment by illiteratewriter** [comment]
I was wondering the same, I actually merged `master` in local and pushed it but it is still showing merge conflicts also.

**Review by davidacevedo — APPROVED** [review]
Looks good, took me a bit to figure out what was going on, so sorry for the delay!

**Pull request merged**